test/aries-js-worker/test/vdri/vdri.js

```diff
@@ -54,4 +76,62 @@ describe("VDRI", function () {
         )
         done()
     })
+
+    it("Alice stores the did received from the college", async function (done) {
```

Comment:
Not sure if Alice received DID from the college. Usually it's the user who generates the DID.